Travel Scheduler/View Controllers/ScheduleViewController.m

```diff
@@ -139,6 +143,33 @@ - (void)scheduleViewSetup
     [self dateCell:nil didTap:removeTime(self.scheduleMaker.startDate)];
 }
 
+#pragma mark - Create Map Button
+
+- (void) createGoToMapButton{
+    self.buttonToGoToMap = [UIButton buttonWithType:UIButtonTypeCustom];
+    self.buttonToGoToMap.backgroundColor = [UIColor whiteColor];
+    [self.buttonToGoToMap setFrame:CGRectZero];
+    //[self.buttonToGoToMap setBackgroundImage:[UIImage imageNamed:@"map_icon"] forState: UIControlStateNormal];
+    [self.buttonToGoToMap setTitle:@"Map" forState:UIControlStateNormal];
+    self.buttonToGoToMap.backgroundColor = [UIColor blueColor];
+    self.buttonToGoToMap.layer.cornerRadius = 10;
+    self.buttonToGoToMap.clipsToBounds = YES;
+    [self.view addSubview:self.buttonToGoToMap];
+    [self.buttonToGoToMap addTarget: self action: @selector(mapButtonClicked:) forControlEvents: UIControlEventTouchUpInside];
+}
+
+- (void)mapButtonClicked: (id)sender
+{
+    if(self.dayPath.count > 0) {
```

Comment:
Is there a situation where self.dayPath.count is not grater than 0? What happens then? 
(not necessary, but it would be nice to think about that)

Reply:
The case check is to ensure that if there are no places for that day no map shows up. Angela has a check for that as well and she even makes those dates not selectable. However, this is just another check.